setupApp.py

```diff
@@ -56,27 +55,6 @@ def _findModules(root, extensions, skip, parent=""):
                 else:
                     yield moduleName
 
-def getStdLibModules():
```

Comment:
Why this change?

Reply:
oh will revert... this is result of research done with the latest py2app and embedding std module. It seems not be necessary to explicitly list every module (will be a different PR)